vMenu/MainMenu.cs

```diff
@@ -118,6 +128,26 @@ public MainMenu()
             }
             #endregion
 
+            #region loading language
+            LanguageData = ConfigManager.GetLanguages();
+
+            string[] LanguageType = { "american", "french", "german", "italian", "spanish", "portuguese", "polish", "russian", "korean", "chinesetraditional", "japanese", "mexican", "chinesesimplified" };
+            string LanguageName = "";
+            
+            try {
```

Comment:
Please check if the value is in there so you don't need to use a try/catch here. Try/Catch should only be used if really necessary.